llpc/unittests/standaloneCompiler/testInputUtils.cpp

```diff
@@ -205,6 +205,41 @@ TEST(InputUtilsTest, IsSpirvBinaryFile) {
 
 TEST(InputUtilsTest, IsGlslShaderFile) {
   // Based on https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/.
+  //
+  // Copyright (C) 2002-2005 3Dlabs Inc. Ltd.
+  // Copyright (C) 2012-2013 LunarG, Inc.
```

Comment:
This test only uses the website for the list of all extensions, not for the implementation. I don't think this requires the license, but I'm not a lawyer.

Reply:
well, not added by me...